bindings/pyroot/pythonizations/test/roofit/rooworkspace.py

```diff
@@ -54,9 +54,6 @@ def test_set_item_using_string(self):
     def test_set_item_using_dictionary(self):
         
         # Import Factory Expressions to new workspace
-        etc_dir = ROOT.std.string(ROOT.TROOT.GetEtcDir().Data())
```

Comment:
The comment with `# Import Factory Expressions to new workspace` is meaningless now. Can you remove it too please?

Also, the file is not correctly formatted. Can you format it with `black --line-length=120` in the same commit? Thanks!

Reply:
Sure, I've changed it!